README.md

```diff
@@ -1,4 +1,13 @@
 American-Gut
 ============
 
-American Gut open-access data and IPython notebooks
+American Gut open-access code and IPython notebooks
+
+A note about Data
+---
+
+American Gut sequences and metadata are deposited in [The European Bioinformatics Institute](http://www.ebi.ac.uk/ena) as ERP012803.
+
+The latest OTU tables and precalculated diversity comparisons generated by the primary processing notebook set can be found at [ftp://ftp.microbio.me/AmericanGut/](ftp://ftp.microbio.me/AmericanGut/).
```

Comment:
This was deliberate; As the project grows, I think its better to let people select the dataset they wish to use. I think its easier when presented with all the data, over being forced into the latest set.